commands/management/add_mongodb.go

```diff
@@ -86,12 +91,34 @@ func (cmd *addMongoDBCommand) GetSocket() string {
 	return cmd.Socket
 }
 
+func loadCertificate(file string) (string, error) {
```

Comment:
A useless comment: we are not doing any special treatment to the data being read here.
It could be a certificate or a key file so, maybe, the function name could be `loadFile`